Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1794,6 +1803,11 @@ open class NavigationMapView: UIView {
                    layerInfo.type.rawValue != "symbol",
                    let sourceLayer = mapView.mapboxMap.style.layerProperty(for: layerInfo.id, property: "source-layer").value as? String,
                    !sourceLayer.isEmpty {
+                    if layerInfo.type.rawValue == "circle",
+                       let isPersistentCircle = try? mapView.mapboxMap.style.isPersistentLayer(id: layerInfo.id),
+                       !isPersistentCircle {
```

Comment:
Is this logic correct? I would’ve assumed that any kind of annotation or data visualization atop the map would use a persistent circle layer, so a persistent circle layer would need to be skipped, but a non-persistent circle layer would be treated as the target layer.